spec/factories/foodbank_text.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :foodbank_text do
+    fb_id { 0 }
+    order { 50 }
+    date_added { 'CURRENT_TIMESTAMP' }
+    added_by { 0 }
+    status_id { 1 }
+    text { Faker::Name.name }
+    link_text { Faker::Name.name }
```

Comment:
Curious why you used Faker::Name instead of Faker::Text?  Not sure if Name includes special UTF characters.